app/controllers/messages_controller.rb

```diff
@@ -6,11 +6,16 @@ def index
     @messages = Message.all
   end
   
+  def new
+    @message = Message.new
+  end
+
   def create
     if Message.between(params[:sender_id], params[:recipient_id]).present?
```

Comment:
Min this is a good point, instead of using params[:sender_id] you should use current_user.id so that a user can only send messages as himself and not be able to impersonate sending messages as someone else.